.circleci/config.yml

```diff
@@ -0,0 +1,41 @@
+version: 2.1
+
+workflows:
+  main:
+    jobs:
+      - lint
+      - unit-tests
+      - build
+
+jobs:
+  lint:
+    docker:
+      - image: circleci/node:12
+    steps:
+      - checkout
+      - run:
+          name: "Lint Code"
+          command: |
+            yarn install --frozen-lockfile
```

Comment:
You can use the `node/install packages` command built into the orb you're using and it should run `yarn install --frozen-lockfile` by default: https://github.com/mozilla/treeherder/blob/master/.circleci/config.yml#L11